src/auth/utils/useSignIn.ts

```diff
@@ -0,0 +1,20 @@
+import { axiosInstance } from '@/src/commons/util/axiosInstance';
+import { useAsync } from '@/src/commons/util/useAsync';
+import { Token } from '../type';
+
+export const useSignIn = (email: string, password: string) => {
+  const signIn = () =>
+    axiosInstance.post<{ data: Token }>('sign-in', {
+      email: email,
+      password: password,
```

Comment:
여기는 그냥 간단하게 아래처럼 쓸 수 있을듯합니다~
```suggestion
      email,
      password,
```

Reply:
key value가 같으면 줄일 수 있었던 걸 잊었네요! 수정하겠습니다.